packages/config/config/devices/0x013c/pst02b.json

```diff
@@ -38,37 +37,183 @@
 			"maxValue": 100,
 			"defaultValue": 100
 		},
-		"5": {
-			"label": "Operation Mode",
-			"description": "Mode of operation and enabled multisensor functions",
+		"5[0x02]": {
+			"label": "Test Mode",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 127,
-			"defaultValue": 0
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"5[0x08]": {
+			"label": "Temperature Scale",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Fahrenheit",
+					"value": 0
+				},
+				{
+					"label": "Celsius",
+					"value": 1
+				}
+			]
+		},
+		"5[0x10]": {
+			"label": "Illumination Report on Trigger",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 1
+				},
+				{
+					"label": "Enable",
+					"value": 0
+				}
+			]
+		},
+		"5[0x20]": {
+			"label": "Temperature Report on Trigger",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 1
+				},
+				{
+					"label": "Enable",
+					"value": 0
+				}
+			]
 		},
 		"6": {
-			"label": "Multisensor Function Switch",
-			"description": "Enable or disable functions of the multisensor.",
+			"label": "PIR Integrate Illumination",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 127,
-			"defaultValue": 4
+			"maxValue": 2,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 2
+				},
+				{
+					"label": "Enable",
+					"value": 0
+				}
+			]
 		},
-		"7": {
-			"label": "Customer Function",
-			"description": "Enable or disable functions of the multisensor. Not applicable for TSP01.",
+		"7[0x02]": {
+			"label": "Send Motion Off Report",
 			"valueSize": 1,
 			"minValue": 0,
-			"maxValue": 127,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"7[0x04]": {
+			"label": "PIR Super Sensitivity Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
```

Comment:
This doesn't match the old default value of 0.

Reply:
According to this document, the default value for parameter 7 is 4: https://doc.eedomus.com/files/PST02_Manual-2014-09-17.pdf